### ISSUE TITLE
Cannot sign notification service using match

### ISSUE THREAD

**Issue by batkov** [fastlane/fastlane#14868]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Cannot sign notification service using match
I see both:
`Detected provisioning profile mapping: {:"com.<appname>.stage"=>"match AdHoc com.<appname>.stage", :"com.<appname>.stage<appname>notificationservice"=>"match AdHoc com.<appname>.stage.<appname>notificationservice"}`
and
`[12:16:10]: ▸ ❌  error: No profiles for 'com.<appname>.stage.<appname>notificationservice' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.<appname>.stage.<appname>notificationservice'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target '<appname>NotificationService')`

##### Command executed
match
Gym 

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> appname username$ sudo bundle exec fastlane beta
Password:
[✔] 🚀 
[12:21:39]: ------------------------------
[12:21:39]: --- Step: default_platform ---
[12:21:39]: ------------------------------
[12:21:39]: Driving the lane 'ios beta' 🚀
[12:21:39]: -------------------
[12:21:39]: --- Step: match ---
[12:21:39]: -------------------
[12:21:39]: Successfully loaded '/Users/username/Projects/appname/iPhone/iPhone/appname/fastlane/Matchfile' 📄

+----------------+-------------------------------------------------------+
|              Detected Values from './fastlane/Matchfile'               |
+----------------+-------------------------------------------------------+
| username       | email@email.com                                       |
| git_url        | https://github.com/app/certs  						|
|                | es.git                                                |
| storage_mode   | git                                                   |
| type           | adhoc                                                 |
| app_identifier | ["com.appname.stage",                         |
|                | "com.appname.stage.appnamenotificationservice"  |
|                | ]                                                     |
+----------------+-------------------------------------------------------+

+-----------------------+-------------------------------------------------------+
|                           Summary for match 2.125.2                           |
+-----------------------+-------------------------------------------------------+
| app_identifier        | ["com.appname.stage",                         |
|                       | "com.appname.stage.appnamenotificationservice"  |
|                       | ]                                                     |
| type                  | adhoc                                                 |
| readonly              | false                                                 |
| username              | email@email.com                                       |
| team_id               | 6Q7KMYUX2E                                            |
| storage_mode          | git                                                   |
| git_url               | https://github.com/app/certs  						|
|                       | es.git                                                |
| git_branch            | master                                                |
| shallow_clone         | false                                                 |
| clone_branch_directly | false                                                 |
| keychain_name         | login.keychain                                        |
| force                 | false                                                 |
| force_for_new_devices | false                                                 |
| skip_confirmation     | false                                                 |
| skip_docs             | false                                                 |
| platform              | ios                                                   |
| verbose               | false                                                 |
+-----------------------+-------------------------------------------------------+

[12:21:39]: Cloning remote git repo...
[12:21:39]: If cloning the repo takes too long, you can use the `clone_branch_directly` option in match.
[12:21:42]: 🔓  Successfully decrypted certificates repo
[12:21:42]: Verifying that the certificate and profile are still valid on the Dev Portal...
Available session is not valid any more. Continuing with normal login.
[12:21:47]: Installing certificate...

+-------------------+-----------------------------------------------------------+
|                             Installed Certificate                             |
+-------------------+-----------------------------------------------------------+
| User ID           | 6Q7KMYUX2E                                                |
| Common Name       | iPhone Distribution: LabStyle Innovation Ltd (6Q7KMYUX2E) |
| Organisation Unit | 6Q7KMYUX2E                                                |
| Organisation      | LabStyle Innovation Ltd                                   |
| Country           | US                                                        |
| Start Datetime    | 2019-06-06 05:16:52 UTC                                   |
| End Datetime      | 2020-06-05 05:16:52 UTC                                   |
+-------------------+-----------------------------------------------------------+

[12:21:48]: Installing provisioning profile...
[12:21:49]: Installing provisioning profile...

+---------------------+------------------------------------+------------------------------------+
|                                Installed Provisioning Profile                                 |
+---------------------+------------------------------------+------------------------------------+
| Parameter           | Environment Variable               | Value                              |
+---------------------+------------------------------------+------------------------------------+
| App Identifier      |                                    | com.appname.stage          |
| Type                |                                    | adhoc                              |
| Platform            |                                    | ios                                |
| Profile UUID        | sigh_com.appname.stage_ad  | ad34945e-42ab-4d28-aef8-4e0e08f54  |
|                     | hoc                                | 2bd                                |
| Profile Name        | sigh_com.appname.stage_ad  | match AdHoc                        |
|                     | hoc_profile-name                   | com.appname.stage          |
| Profile Path        | sigh_com.appname.stage_ad  | /Users/username/Library/MobileD  |
|                     | hoc_profile-path                   | evice/Provisioning                 |
|                     |                                    | Profiles/ad34945e-42ab-4d28-aef8-  |
|                     |                                    | 4e0e08f542bd.mobileprovision       |
| Development Team ID | sigh_com.appname.stage_ad  | 6Q7KMYUX2E                         |
|                     | hoc_team-id                        |                                    |
+---------------------+------------------------------------+------------------------------------+

+---------------------+------------------------------------+------------------------------------+
|                                Installed Provisioning Profile                                 |
+---------------------+------------------------------------+------------------------------------+
| Parameter           | Environment Variable               | Value                              |
+---------------------+------------------------------------+------------------------------------+
| App Identifier      |                                    | com.appname.stage.appnameno  |
|                     |                                    | tificationservice                  |
| Type                |                                    | adhoc                              |
| Platform            |                                    | ios                                |
| Profile UUID        | sigh_com.appname.stage.appname     | 9d4f8eca-9929-47fd-a9d5-e7d8111a0  |
|                     | notificationservice_adhoc          | 175                                |
| Profile Name        | sigh_com.appname.stage.appnameno   | match AdHoc                        |
|                     | notificationservice_adhoc_prof     | com.appname.stage.appnameno  |
|                     | ile-name                           | tificationservice                  |
| Profile Path        | sigh_com.appname.stage.appname     | /Users/username/Library/MobileD  |
|                     | notificationservice_adhoc_prof     | evice/Provisioning                 |
|                     | ile-path                           | Profiles/9d4f8eca-9929-47fd-a9d5-  |
|                     |                                    | e7d8111a0175.mobileprovision       |
| Development Team ID | sigh_com.appname.stage.appname.    | 6Q7KMYUX2E                         |
|                     | notificationservice_adhoc_team     |                                    |
|                     | -id                                |                                    |
+---------------------+------------------------------------+------------------------------------+

[12:21:50]: All required keys, certificates and provisioning profiles are installed 🙌
[12:21:50]: Setting Provisioning Profile type to 'ad-hoc'
[12:21:50]: -----------------
[12:21:50]: --- Step: gym ---
[12:21:50]: -----------------
[12:21:50]: $ xcodebuild -showBuildSettings -workspace appname.xcworkspace -scheme appname
[12:21:53]: Command timed out after 3 seconds on try 1 of 4, trying again with a 6 second timeout...
[12:21:57]: Detected provisioning profile mapping: {:"com.appname.stage"=>"match AdHoc com.appname.stage", :"com.appname.stage.appnamenotificationservice"=>"match AdHoc com.appname.stage.appnamenotificationservice"}

+-------------------------------------------------------+-------------------------------------------------------+
|                                            Summary for gym 2.125.2                                            |
+-------------------------------------------------------+-------------------------------------------------------+
| workspace                                             | appname.xcworkspace                                     |
| scheme                                                | appname                                                 |
| export_method                                         | ad-hoc                                                |
| clean                                                 | true                                                  |
| export_options.provisioningProfiles.com.appname	    | match AdHoc com.appname.stage                 |
| .stage                                           		|                                                       |
| export_options.provisioningProfiles.com.appname  	| match AdHoc                                           |
| .stage.appnamenotificationservice                   | com.appname.stage.appnamenotificationservice    |
| destination                                           | generic/platform=iOS                                  |
| output_name                                           | appname                                                 |
| build_path                                            | /Users/username/Library/Developer/Xcode/Archives/2  |
|                                                       | 019-06-07                                             |
| output_directory                                      | .                                                     |
| silent                                                | false                                                 |
| skip_package_ipa                                      | false                                                 |
| result_bundle                                         | false                                                 |
| buildlog_path                                         | ~/Library/Logs/gym                                    |
| skip_profile_detection                                | false                                                 |
| xcode_path                                            | /Applications/Xcode.app                               |
+-------------------------------------------------------+-------------------------------------------------------+

[12:21:58]: $ set -o pipefail && xcodebuild -workspace appname.xcworkspace -scheme appname -destination 'generic/platform=iOS' -archivePath /Users/username/Library/Developer/Xcode/Archives/2019-06-07/appname\ 2019-06-07\ 12.21.58.xcarchive clean archive | tee /Users/username/Library/Logs/gym/appname-appname.log | xcpretty
[12:22:00]: ▸ Clean Succeeded
[12:22:06]: ▸ ❌  error: No profiles for 'com.appname.stage.appnamenotificationservice' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.appname.stage.appnamenotificationservice'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target 'appnameNotificationService')
[12:22:06]: ▸ ❌  error: No profiles for 'com.appname.stage' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.appname.stage'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target 'appname')
[12:22:06]: ▸ The iOS deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'AFNetworking')
[12:22:06]: ▸ The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'Amplitude-iOS')
[12:22:06]: ▸ The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'FLAnimatedImage')
[12:22:06]: ▸ The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'SDWebImage')
[12:22:06]: ▸ The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'AppsFlyerFramework')
[12:22:06]: ▸ The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'GoogleUtilities')
[12:22:06]: ▸ The iOS deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'nanopb')
[12:22:06]: ▸ The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'GoogleAppMeasurement')
[12:22:06]: ▸ The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'GTMSessionFetcher')
[12:22:06]: ▸ The iOS deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'leveldb-library')
[12:22:06]: ▸ The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'Protobuf')
[12:22:06]: ▸ The iOS deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'LPMessagingSDK')
[12:22:06]: ▸ The iOS deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'MSWeakTimer')
[12:22:06]: ▸ The iOS deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'NXOAuth2Client')
[12:22:06]: ▸ The iOS deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'TMCache')
[12:22:06]: ▸ The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'SHSPhoneComponent')
[12:22:06]: ▸ The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'libPhoneNumber-iOS')
[12:22:06]: ▸ ** ARCHIVE FAILED **
▸ Clean Succeeded

❌  error: No profiles for 'com.appname.stage.appnamenotificationservice' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.appname.stage.appnamenotificationservice'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target 'appnameNotificationService')

❌  error: No profiles for 'com.appname.stage' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.appname.stage'. Automatic signing is disabled and unable to generate a profile. To enable automatic signing, pass -allowProvisioningUpdates to xcodebuild. (in target 'appname')

The iOS deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'AFNetworking')
The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'Amplitude-iOS')
The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'FLAnimatedImage')
The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'SDWebImage')
The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'AppsFlyerFramework')
The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'GoogleUtilities')
The iOS deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'nanopb')
The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'GoogleAppMeasurement')
The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'GTMSessionFetcher')
The iOS deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'leveldb-library')
The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'Protobuf')
The iOS deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'LPMessagingSDK')
The iOS deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'MSWeakTimer')
The iOS deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'NXOAuth2Client')
The iOS deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'TMCache')
The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'SHSPhoneComponent')
The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'libPhoneNumber-iOS')
** ARCHIVE FAILED **
[12:22:06]: Exit status: 65

+---------------+-------------------------+
|            Build environment            |
+---------------+-------------------------+
| xcode_path    | /Applications/Xcode.app |
| gym_version   | 2.125.2                 |
| export_method | ad-hoc                  |
| sdk           | iPhoneOS12.2.sdk        |
+---------------+-------------------------+

[12:22:06]: ▸ Build system information
[12:22:06]: ▸ warning: The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'SHSPhoneComponent')
[12:22:06]: ▸ Build system information
[12:22:06]: ▸ warning: The iOS deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.2.99. (in target 'libPhoneNumber-iOS')
[12:22:06]: 
[12:22:06]: ⬆️  Check out the few lines of raw `xcodebuild` output above for potential hints on how to solve this error
[12:22:06]: 📋  For the complete and more detailed error log, check the full log at:
[12:22:06]: 📋  /Users/username/Library/Logs/gym/appname-appname.log
[12:22:06]: 
[12:22:06]: Looks like fastlane ran into a build/archive error with your project
[12:22:06]: It's hard to tell what's causing the error, so we wrote some guides on how
[12:22:06]: to troubleshoot build and signing issues: https://docs.fastlane.tools/codesigning/getting-started/
[12:22:06]: Before submitting an issue on GitHub, please follow the guide above and make
[12:22:06]: sure your project is set up correctly.
[12:22:06]: fastlane uses `xcodebuild` commands to generate your binary, you can see the
[12:22:06]: the full commands printed out in yellow in the above log.
[12:22:06]: Make sure to inspect the output above, as usually you'll find more error information there
[12:22:06]: 
+------------------------------------+-------------------------------------------------------+
|                                        Lane Context                                        |
+------------------------------------+-------------------------------------------------------+
| DEFAULT_PLATFORM                   | ios                                                   |
| PLATFORM_NAME                      | ios                                                   |
| LANE_NAME                          | ios beta                                              |
| SIGH_PROFILE_TYPE                  | ad-hoc                                                |
| MATCH_PROVISIONING_PROFILE_MAPPING | {"com.appname.stage"=>"match AdHoc            |
|                                    | com.appname.stage",                           |
|                                    | "com.appname.stage.appnamenotificationservice"  |
|                                    | =>"match AdHoc                                        |
|                                    | com.appname.stage.appnamenotificationservice"}  |
+------------------------------------+-------------------------------------------------------+
[12:22:06]: Error building the application - see the log above

+------+------------------+-------------+
|           fastlane summary            |
+------+------------------+-------------+
| Step | Action           | Time (in s) |
+------+------------------+-------------+
| 1    | default_platform | 0           |
| 2    | match            | 11          |
| 💥   | gym              | 15          |
+------+------------------+-------------+

[12:22:06]: fastlane finished with errors

[!] Error building the application - see the log above</pre>
</details>

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.14.5                                     |
| Ruby                        | 2.4.2                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.20.1 (Apple Git-117)          |
| Installation Source         | /usr/local/bin/fastlane                     |
| Host                        | Mac OS X 10.14.5 (18F132)                   |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.2_1/lib          |
| OpenSSL Version             | OpenSSL 1.0.2l  25 May 2017                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 10.2.1                                      |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:ios)

platform :ios do
  desc "Push a new beta build to Hockey"
  lane :beta do
	match(app_identifier: ["com.<appname>.stage", "com.<appname>.stage.<appname>notificationservice"], type:"adhoc")

    gym(workspace: "<appname>.xcworkspace", 
    	scheme: "<appname>", 
    	export_method: "ad-hoc",
    	clean:true

	  )
    hockey(api_token: "token")
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("com.<appname>.stage") # The bundle identifier of your app
apple_id("email@email.com") # Your Apple email address

itc_team_id("teamid") # App Store Connect Team ID
team_id("teamid") # Developer Portal Team ID

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.125.2 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.9.0        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.6.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.5.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.6.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.7.0        |
| tty-cursor                | 0.7.0        |
| tty-spinner               | 0.9.1        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.64.0       |
| faraday                   | 0.15.4       |
| unf_ext                   | 0.0.7.6      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20180417 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.5        |
| gh_inspector              | 1.1.3        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.3        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.7.2        |
| bundler                   | 2.0.1        |
| faraday_middleware        | 0.13.1       |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.5        |
| jwt                       | 2.1.0        |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mime-types-data           | 3.2019.0331  |
| mime-types                | 3.2.2        |
| multi_json                | 1.13.1       |
| signet                    | 0.11.0       |
| memoist                   | 0.16.0       |
| os                        | 1.0.1        |
| googleauth                | 0.6.7        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.23.9       |
| google-cloud-core         | 1.3.0        |
| digest-crc                | 0.4.1        |
| google-cloud-storage      | 1.16.0       |
| emoji_regex               | 1.0.1        |
| openssl                   | 2.0.5        |
| json                      | 2.2.0        |
| io-console                | 0.4.6        |
| CFPropertyList            | 3.0.0        |
| psych                     | 2.2.2        |
</details>

*generated on:* **2019-06-07**
</details>

**Comment by janpio** [comment]
Is Adhoc the same as Development in this context? Because first it says:

`"com.<appname>.stage<appname>notificationservice"=>"match AdHoc com.<appname>.stage.<appname>notificationservice"` 

and then

`Xcode couldn't find any iOS App Development provisioning profiles matching 'com.<appname>.stage.<appname>notificationservice'`

**Comment by batkov** [comment]
Hi, @janpio 
I found the issue is to run tools via `sudo`, so this particular issue was solved by updating `chown` and `chmod` on `permission denied` folders.
But afterwards I tried to setup another project with all permissions already granted and still have similar issues.
But now it was issue with keychain, so I used:
`bundle exec fastlane run unlock_keychain path:$KEYCHAIN_NAME password:$KEYCHAIN_PASS`

Probably it would be good to put something to troubleshooting article.
